src/cmd/services/m3dbnode/config/config.go

```diff
@@ -133,6 +143,16 @@ type DBConfiguration struct {
 	WriteNewSeriesAsync bool `yaml:"writeNewSeriesAsync"`
 }
 
+// Validate validates the Configuration. We use this method to validate
+// fields where the validator package falls short.
+func (c DBConfiguration) Validate() error {
+	if err := c.Filesystem.Validate(); err != nil {
```

Comment:
Can this be simplified to `return c.Filesystem.Validate()`?

Reply:
same comment as above